vue-components/stories/Lookup.stories.ts

```diff
@@ -75,16 +93,25 @@ export function all(): Component {
 				search: '',
 				selectedItem: null,
 				visibleItems: null,
+				menuVegetableItems: vegetableItems,
 			};
 		},
 		computed: {
-			menuItems(): MenuItem[] {
-				return vegetableItems.filter( ( item ) => item.label.includes( this.search ) );
+			menuItems: {
+				get(): MenuItem[] {
+					return this.menuVegetableItems.filter( ( item ) => item.label.includes( this.search ) );
+				},
+				set( newMenuItems: MenuItem[] ): void {
+					this.menuVegetableItems = newMenuItems;
+				},
 			},
 		},
 		methods: {
 			onScroll( firstIndex: number, lastIndex: number ): void {
 				this.visibleItems = { firstIndex, lastIndex };
+				if ( lastIndex === this.menuItems.length - 1 ) {
+					this.menuItems = [ ...vegetableItems, ...addOnScroll ];
+				}
 			},
 		},
 		template: `
```

Comment:
I think we could add the instructions somewhere below here (GitHub won’t let me comment on the right line :rage:), something like:

```html
<p>
Try pressing <kbd>a</kbd> to see several search results. Some additional results will load when you scroll.
</p>
```